README_dev.md

```diff
@@ -149,7 +149,7 @@ npm run test:local -- testSuiteName
       },
       "database": { "driver": "lokijs", "folder": "./" },
       "logging": { "categories": { "default": { "appenders": ["console"] } } },
-      "infrastructure": { "httpServer": { "apiKey": "xxx", "port": 8080 } },
+      "infrastructure": { "httpServer": { "apiKey": "", "port": 8080 } },
```

Comment:
Maybe use `"..."` instead of `""` as placeholder for the `"apiKey"` as it is done for the other fields (`"baseUrl"`, `"platformClientId"`, `"platformClientSecret"`).

Reply:
if it was a placeholder I would have used `<...>`

Reply:
> It disables the API key.

Good to know!

> if it was a placeholder I would have used `<...>`

I would have appreciated that too (even if the example unfortunately uses `"..."` as a placeholder instead of `"<...>"` so far).

Reply:
done.